.github/workflows/central_release.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish package to the Maven Central Repository
+on:
+  release:
+    types: [created]
+jobs:
+  publish:
+    environment:
+      name: Central Repository Deployment
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+      - name: Publish Testify Gradle plugin
+        run: ./gradlew Plugin:uploadArchives
+        env:
+          OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
+          ORG_GRADLE_PROJECT_signingKeyId: ${{ secrets.OSSRH_GPG_SECRET_KEY_ID }}
+          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.OSSRH_GPG_SECRET_KEY }}
+          ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.OSSRH_GPG_SECRET_KEY_PASSWORD }}
+      - name: Publish Testify Library
```

Comment:
Nitpick, but really the library should be published first. The plugin depends on the library, so if anything were to go wrong with publishing the library, I'd want this to fail and not publish the plugin.

Reply:
Since they are published to a staging repository first, then manually promoted, this doesn't really matter. But I'll swap em' in case we move to fully automatic publishing.